must-gather/collection-scripts/gather_clusterscoped_resources

```diff
@@ -64,3 +64,20 @@ for command_desc in "${commands_desc[@]}"; do
      COMMAND_OUTPUT_FILE=${BASE_COLLECTION_PATH}/cluster-scoped-resources/oc_output/desc_${command_desc// /_}
     { oc describe "${command_desc}"; } >> "${COMMAND_OUTPUT_FILE}"
 done
+
+kenel_collection() {
+        oc debug nodes/"${node}" --to-namespace="${ns}" -- bash -c "sleep 5m" &
+        while [ "$(oc get pods -n openshift-storage | grep "${node//./}-debug" | awk '{print $2}')" != "1/1" ] ; do
+            sleep 3
+            echo "waiting for the debug pod to be in ready state"
+        done
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":var/log/rook "${KENEL_OUTPUT_DIR}"
```

Comment:
Upon using this command on my machine, I got an error saying 
```
oc rsync ip-10-0-142-177us-east-2computeinternal-debug:/var/ ./kernel-logs
WARNING: cannot use rsync: rsync not available in container
```
And it just stuck here. Can this might be my internet issue?.

Reply:
This worked, seems like it was my internet issue.